### PR TITLE
Upgrade to v2.69.0 of the AzureRM Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #928﻿

### CODE REVIEW

File: provider/resource.go [hunk 1]
```diff
@@ -45,99 +45,100 @@ const (
...
-	azureSearch                = "Search"                // Search
-	azureSignalr               = "SignalR"               // SignalR
-	azureSiteRecovery          = "SiteRecovery"          // SiteRecovery
-	azureSQL                   = "Sql"                   // SQL
-	azureStack                 = "Stack"                 // Stack
-	azureStorage               = "Storage"               // Storage
-	azureStreamAnalytics       = "StreamAnalytics"       // StreamAnalytics
-	azureSynapse               = "Synapse"               // Synapse
-	azureWaf                   = "Waf"                   // WAF
+	activeDirectoryDomainService = "ActiveDirectoryDomainService" // ActiveDirectoryDomainService
```
Comment (mikhailshilkov) [inline provider/resource.go:48]: Is it not just "DomainServices" as in https://github.com/Azure/azure-rest-api-specs/tree/master/specification/domainservices/resource-manager ?

File: provider/resource.go [hunk 2]
```diff
@@ -1688,6 +1703,7 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_bot_web_app":               {Tok: azureResource(azureBot, "WebApp")},
 			"azurerm_bot_channel_ms_teams":      {Tok: azureResource(azureBot, "ChannelTeams")},
 			"azurerm_bot_channel_directline":    {Tok: azureResource(azureBot, "ChannelDirectLine")},
+			"azurerm_bot_channel_web_chat":      {Tok: azureResource(azureBot, "WebChat")},
```
Comment (mikhailshilkov) [inline provider/resource.go:1706]: Should it be `ChannelWebChat`?

Comment (stack72) [reply provider/resource.go:1706]: Great catch! thanks @mikhailshilkov

File: provider/resource.go (hunk 1 above)
Comment (mikhailshilkov) [reply provider/resource.go]: I think plural `domainservices` would be a better match than `domainservice`

**Review by mikhailshilkov — APPROVED** [review]
LGTM

**Pull request merged**